src/Service.Host/client/src/components/reports/SernosUsedOnInvoice.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment } from 'react';
+import { Grid } from '@material-ui/core';
+import {
+    ReportTable,
+    Loading,
+    Title,
+    BackButton,
+    ErrorCard
+} from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import Page from '../../containers/Page';
+
+function Report({ reportData, loading }) {
+    if (loading) {
+        return <Loading />;
+    }
+    return (
+        <Fragment>
+            {reportData && reportData.results.length > 0 ? (
+                <ReportTable
+                    showRowTitles={false}
+                    reportData={reportData}
+                    showTotals={false}
+                    title={reportData ? reportData.title.displayString : null}
+                    showTitle={false}
+                />
+            ) : (
+                <ErrorCard errorMessage="No Serial Numbers Found For Specified Parameters" />
+            )}
+        </Fragment>
+    );
+}
+
+function SernosUsedOnInvoice({ reportData, loading, history }) {
+    const handleBackClick = () => {
+        history.push('/products/reports/sernos-used-on-invoice');
+    };
+    return (
+        <Page>
+            <Grid container spacing={24} justify="center">
```

Comment:
Just from what i've been working on my branch - i'm passing errorMessage into here from the container which is retrieved from getSingleErrorMessage(state).. and i'm placing an error card at the top of forms

Reply:
👍